.platform.app.yaml

```diff
@@ -152,6 +152,9 @@ crons:
   d7db:
     spec: '0 1 * * *'
     cmd: 'migrate-scripts/sync-db.sh $D7_PROJECT_ID $D7_PROJECT_ENV'
+  sync_files:
+    spec: '30 0 * * *'
+    cmd: 'migrate-scripts/sync-private-files.sh $D7_PROJECT_ID $D7_PROJECT_ENV'
```

Comment:
Need this to periodically fetch the private files from D7, as we can't use the HTTP approach to download them like with public files; Drupal handles each request for private files and we'd otherwise get HTTP 403.